js/index.js

```diff
@@ -0,0 +1,20 @@
+const currentDate = new Date();
+const thisYear = currentDate.getFullYear(); 
+
+const footer = document.createElement("footer");
+const copyright = document.createElement("p"); 
+copyright.innnerHTML = `<small>Helen Abera &copy; ${thisYear}</small>`;
+
+footer.appendChild(copyright);
+document.body.appendChild(footer); 
+
+let skills = ["Project Management", "Health Care Admin", "JavaScript", "CSS", "HTML"];
+let skillsSection = document.getElementById("skills"); 
+let skillsList = document.createElement("ul"); 
+skillsSection.appendChild(skillsOrder); 
+
+for (let skill of skills) {
+    const skillItem = document.createElement("li");
+    skill.innerText = skillItem;
+    skillsListappendChild(skill); 
```

Comment:
Should append skillItem which is the li element rather than the skill string.

Reply:
Hello Tom - thank you for going through this. I am currently in the process of redoing this project since I changed it a few times via the different folders I had set up via VS code. I think it did more harm than good but I am working on this right now to make sure I cover all my errors and to complete the project properly.